src/lib.rs

```diff
@@ -913,8 +913,8 @@ mod tests {
             assert_eq!(None, iter.next());
         }
 
-        assert_eq!(17, map["a"]);
-        assert_eq!(23, map["b"]);
+        assert_eq!(17, map[&"a"]);
+        assert_eq!(23, map[&"b"]);
```

Comment:
This is actually the result of another bug: https://github.com/rust-lang/rust/issues/23649. We should be able to go back to `map["a"]` once it's fixed.